kotlin/navigate-route-with-rerouting/README.md

```diff
@@ -0,0 +1,71 @@
+3. Set `isReturnRoutes`, `isReturnStops`, and `isReturnDirections` on the parameters to `true`.
+4. Add `Stop`s to the parameters' array of `stops` using `RouteParameters.setStops()`.
+5. Solve the route using `ARouteTask.solveRouteAsync(routeParameters)` to get an `RouteResult`.
+6. Create an `RouteTracker` using the route result, and the index of the desired route to take.
+7. Enable rerouting in the route tracker using `RouteTracker.enableReroutingAsync(reroutingParameters)`. Pass `RouteTracker.ReroutingStrategy.TO_NEXT_WAYPOIN` as the value of `strategy` to specify that in the case of a reroute, the new route goes from present location to next waypoint or stop.
+8. Implement `RouteTracker.trackLocationAsync()` to track the location of the device and update the route tracking status.
+9. Implement `RouteTracker.TrackingStatusChangedListener()` to be notified of `TrackingStatus` changes, and use them to display updated route information. `TrackingStatus` includes a variety of information on the route progress, such as the remaining distance, remaining geometry or traversed geometry (represented by an `Polyline`), or the remaining time (`Double`), amongst others.
+10. You can also query the tracking status for the current `DirectionManeuver` index by retrieving that maneuver from the `Route` and getting its direction text to display in the GUI.
+11. To establish whether the destination has been reached, get the `DestinationStatus` from the tracking status. If the destination status is `DestinationStatus.REACHED` and the `DirectionManeuvers.size` is 1, you have arrived at the destination and can stop routing. If there are several destinations on your route and the remaining destination count is greater than 1, switch the route tracker to the next destination.
+
+## Relevant API
+
+* DestinationStatus
+* DirectionManeuver
+* Location
+* LocationDataSource
+* ReroutingStrategy
+* Route
+* RouteParameters
+* RouteTask
+* RouteTracker
+* RouteTrackerLocationDataSource
+* SimulatedLocationDataSource
+* Stop
+* VoiceGuidance
+
+## Offline data
+
+1. Download the data from [ArcGIS Online](https://arcgisruntime.maps.arcgis.com/home/item.html?id=df193653ed39449195af0c9725701dca).
+2. Extract the contents of the downloaded zip file to disk.
+3. Open your command prompt and navigate to the folder where you extracted the contents of the data from step 1.
+4. Execute the following commands:
+
+`adb push streetmap_SD.tpkx /Android/data/com.esri.arcgisruntime.sample.offlinerouting/files/san_diego/streetmap_SD.tpkx`
+
+`adb push sandiego.geodatabase /Android/data/com.esri.arcgisruntime.sample.offlinerouting/files/san_diego/sandiego.geodatabase`
+
+Link | Local Location
+---------|-------|
+|[San Diego streetmap TPKX](https://arcgisruntime.maps.arcgis.com/home/item.html?id=df193653ed39449195af0c9725701dca)| /Android/data/com.esri.arcgisruntime.sample.offlinerouting/files/streetmap_SD.tpkx |
```

Comment:
we're not using this it looks like (which I think is a good idea). Let's remove from README and metadata